modules/environments/groups.tf

```
@@ -0,0 +1,19 @@
+resource "azuread_group" "groups" {
+  for_each                = local.groups
+  display_name            = each.value.name
+  description             = each.value.description
+  prevent_duplicate_names = false # to be changed, set to false to ensure successful plan before state migration
```

🚫 needs to be changed to true, Any state issues should be imported to fix it (and scripted)
```suggestion
  prevent_duplicate_names = true
```